tester.py

```diff
@@ -0,0 +1,33 @@
+import os
+import subprocess
+import shlex
+import sys
+verbose = len(sys.argv) > 1 and sys.argv[1] == '-v'
+if(verbose):
+    import difflib
+cwd = os.getcwd()
+
+def runTest(folderName):
+    config = subprocess.Popen(shlex.split('python3 initiate.py "'+folderName+'"/config.txt'), cwd=cwd)
```

Comment:
why are the "test 3" neede? because of the space? just get rid of the space

Reply:
Why does that matter? It's also so that it works with more descriptive names for the test folders, like "test Multiple sellers" would work too. Why not?

Reply:
fare enough